config/crd/bases/management.project-flotta.io_edgeworkloads.yaml

```diff
@@ -472,11 +472,78 @@ spec:
                                                 ANDed.
                                               type: object
                                           type: object
+                                        namespaceSelector:
```

Comment:
why did this file changed?

Reply:
This file is generated, I think the change here is because of the controller-gen upgrade

Reply:
Yes :) this file changed only after running `make generate`, after upgrading `controller-gen`